nevergrad/functions/images/core.py

```diff
@@ -205,3 +200,50 @@ def _with_tag(
         assert func._initialization_kwargs is not None
         func._initialization_kwargs["tags"] = tags
         return func
+
+
+class ImageFromPGAN(base.ExperimentFunction):
+    """
+    Creates face images using a GAN from pytorch GAN zoo trained on celebAHQ and optimizes the noise vector of the GAN
+
+    problem_name: the type of problem we are working on.
+    initial_noise: the initial noise of the GAN. It should be of dimension (1, 512). If None, it is defined randomly.
+    use_gpu: whether to use gpus to compute the images
+    scorer: which scorer to use for the images
+    mutable_sigma: whether the sigma should be mutable
+    n_mutations: number of mutations
+    """
+
+    def __init__(self, problem_name: str = "GAN_optimization", initial_noise: np.ndarray = None, use_gpu: bool = True, scorer=Koncept512Loss, mutable_sigma=True, n_mutations=35) -> None:
```

Comment:
problem name does not seem to be used for anything useful, remove it?

Reply:
I had just copied it from the Image class above. I guess it's not really useful.

Reply:
yeah I figured afterwards, it should not have been in the class above in the first class, I had not noticed it :s tiny mistakes tend to propagate a lot :D